### PR TITLE
add helm lint to travis

### PR DESCRIPTION
**Pull request by yagonobre**
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa/273)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,8 +4,14 @@ go:
   - 1.8
   - tip
 
+before_script:
+  - wget https://storage.googleapis.com/kubernetes-helm/helm-v2.5.1-linux-amd64.tar.gz -O /tmp/helm.tar.gz
```
Comment (aguerra) [inline .travis.yml]: curl ftw!

File: .travis.yml [hunk 2]
```diff
@@ -4,8 +4,14 @@ go:
...
+  - tar -xcf /tmp/helm.tar.gz
```
Comment (aguerra) [inline .travis.yml]: I guess mixing x and c doesn't work, but I'm not sure.

Comment (yagonobre) [inline .travis.yml]: yes, you are right

File: .travis.yml [hunk 3]
```diff
@@ -4,7 +4,13 @@ go:
   - 1.8
   - tip
 
+before_script:
+  - wget https://storage.googleapis.com/kubernetes-helm/helm-v2.5.1-linux-amd64.tar.gz
```
Comment (drgarcia1986) [inline .travis.yml]: ayfkm
WGET?!???!?!

Comment (yagonobre) [inline .travis.yml]: sahsuahsuahushaushaushaus

**Comment by yagonobre** [comment]
@aguerra @drgarcia1986 updated

**Pull request merged**